guides/common/modules/proc_renaming-server.adoc

```diff
@@ -33,15 +33,15 @@ Choose one of the following methods:
 ----
 # {project-change-hostname} _new-{project-context}_ \
 --username _My_Username_ \
---password _My_Password_
+--password '_My_Password_'
 ----
 * If your {ProjectServer} is installed with custom SSL certificates:
 +
 [options="nowrap", subs="+quotes,verbatim,attributes"]
 ----
 # {project-change-hostname} _new-{project-context}_ \
 --username _My_Username_ \
---password _My_Password_ \
+--password '_My_Password_' \
```

Comment:
Would double quotes work too? It would be consistent with the options below.

Reply:
It works, but not as well. Single quotes also prevent variable substitution. Concrete example is `$x` in your password. With single quotes that's literally `$x` while with double quotes it will try to look up $x. If it doesn't exist then it is replaced with an empty string.